src/main/kotlin/org/jitsi/jicofo/jibri/BaseJibri.kt

```diff
@@ -65,15 +66,19 @@ abstract class BaseJibri internal constructor(
         TaskPools.ioPool
     )
 
+    private val clientIqHandler: JibriSessionIqHandler = object : JibriSessionIqHandler {
```

Comment:
Why did you do it this way rather than having `BaseJibri` implement `JibriSessionIqHandler`?

Reply:
Because when I add the "service" XMPP connection for jibries I'll need two separate, one for each connection